src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -394,12 +394,29 @@ impl MDARegions {
     }
 }
 
+/// A type representing the actual size of variable length metadata written within this metadata
```

Comment:
The stratisd test failed on something slightly above this, on line 392:
```
error[E0308]: mismatched types
   --> src/engine/strat_engine/backstore/metadata/mda.rs:392:57
    |
392 |                 .map_or_else(|| true, |mda| mda.used != Bytes(0)));
    |                                                         ^^^^^^^^ expected struct `engine::strat_engine::backstore::metadata::mda::MetaDataSize`, found struct `devicemapper::units::Bytes`
    |
    = note: expected type `engine::strat_engine::backstore::metadata::mda::MetaDataSize`
               found type `devicemapper::units::Bytes`
```

Reply:
Strange; this is part of what was added in stratisd PR #1554, but the stratisd test passed for that PR.  I wonder why it's failing now.

Reply:
Interesting, it didn't pass `cargo check` but `make build` compiled just fine. I wonder what's going on... I fixed it.

Reply:
Looks good so far; the stratisd test is still running, but it's progressed past this point.

Reply:
The stratisd test passed.